content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,25 +9,81 @@ weight: 20
-updating, and deleting primary resources via the standard HTTP verbs (POST, PUT, PATCH, DELETE, GET), includes additional subresources for many objects that allow fine grained authorization (such as binding a pod to a node), and can accept and serve those resources in different representations for convenience or efficiency. It also supports efficient change notifications on resources via "watches" and consistent lists to allow other components to effectively cache and synchronize the state of resources.
+## Kubernetes API terminology {#standard-api-terminology}
+
+Kubernetes generally leverages standard RESTful terminology to describe the
+API concepts:
+
+* A *resource type* is the name used in the URL (`pods`, `namespaces`, `services`)
+* All resource types have a concrete representation in JSON (their object schema) which is called a *kind*
+* A list of instances of a resource type is known as a *collection*
+* A single instance of the resource type is called a *resource*
+* Some resource types have one or more *sub-resources*, represented as URI paths below the resource
+
+Most Kubernetes API resource types are
+[objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/#kubernetes-objects):
+they represent a concrete instance of a concept on the cluster, like a
+pod or namespace. A smaller number of API resource types are *virtual* in
+that they often represent operations on objects, rather than objects, such
+as a permission check
+(use a POST with a JSON-encoded body of `SubjectAccessReview` to the
+`subjectaccessreviews` resource), or the `eviction` sub-resource of a Pod
+(used to trigger
+[API-initiated eviction](/docs/concepts/scheduling-eviction/api-eviction/)).
+
+All objects you can create via the API have a unique object name to allow idempotent creation and retrieval, except that virtual resource types may not have unique names if they are not retrievable, or do not rely on idempotency.
+
+### API verbs
+
+Almost all object resource types support the standard HTTP verbs - GET, POST, PUT, PATCH,
+and DELETE. Kubernetes also uses its own verbs, which are often written lowercase to distinguish
+them from HTTP verbs.
+
+Kubernetes uses the term **list** to describe returning a [collection](#collections) of
+resources to distinguish from retrieving a single resource which is usually called
+a **get**. If you sent an HTTP GET request with the `?watch` query parameter,
+Kubernetes calls this a **watch** and not a **get** (see
+[Efficient detection of changes](#efficient-detection-of-changes) for more details).
+
+For PUT requests, Kubernetes internally classifies these as either **create** or **update**
+based on the state of the existing object. An **update** is different from a **patch**; the
+HTTP verb for a **patch** is PATCH.
 
-## Standard API terminology
+{{< note >}}
+The *server-side apply* concept differentiates between `Apply` and `Update` operations. That
```

Comment:
This is confusing, I'm not sure what it's trying to say. SSA is a special kind of PATCH which can either create or update an object.

Reply:
Ah. The previous page didn't mention an **apply** verb so I misunderstood how things work.

Reply:
Is there an exhaustive list of verbs in the Kubernetes API? If that's feasible to document, we could add it (separately) to the website.

Reply:
For k8s specific "verbs" this is only missing APPLY. For general HTTP verbs, we do useful things on at least HEAD, maybe OPTIONS, I can't recall.

Reply:
The trouble with listing verbs is that RBAC adds extra ones (eg **impersonate**). The reader has to know if the verb is used in an RBAC context or for the general API machinery, aside from authnz. I think I won't try to list them all.

I also won't list all the HTTP verbs as the IETF does a great job on that.

Reply:
Anyway, I think the original feedback https://github.com/kubernetes/website/pull/25299#discussion_r677690861 was valid and is fixed.